File: java/src/com/novell/android/yastroid/Module.java
package com.novell.android.yastroid;

public class Module {
	
	private String name;
	private String display;
	private int icon;

	public Module( String name, String display, int icon ) {
		this.name = name;
		this.display = display;
		this.icon = icon;
	}
	
	public String getName() {
		return name;
	}
	
	public String getDisplay() {
		return display;
	}

	public int getIcon() {
		return icon;
	}

}

File: mono/YaSTroid/YaSTroid/Module.cs

namespace YaSTroid
{
	public class Module
	{
		private string name;
		private string display;
		private int icon;

		public Module( string name, string display, int icon )
		{
			this.name = name;
			this.display = display;
			this.icon = icon;
		}
		
		public string getName()
		{
			return name;
		}
		
		public string getDisplay()
		{
			return display;
		}

		public int getIcon()
		{
			return icon;
		}
	}
}
